src/packages/@ncigdc/containers/explore/ExplorePage.js

```diff
@@ -193,6 +121,13 @@ const ExplorePageComponent = ({
   return (
     <SearchPage
       className="test-explore-page"
+      ControlledAccess={DISPLAY_DAVE_CA && [
```

Comment:
I'm curious why this is an array?

Reply:
in case we later on add more components there, removing the modal... bad idea?

Reply:
oh ok, makes sense. I thought this was some fancy boolean logic

Reply:
hahah I wish I was fancy.